### ISSUE TITLE
Feature Request: Support context menu on tab or background for the current page 

### ISSUE THREAD

**Issue by paulidavey** [andy-portmen/open-in#60]
TLDR :  Add "Open in " to the the tab or page background context menus

Like many I use FF and Chrome for different purposes and use both addons.

I often find I open a link **_and then_** realise that I want to be working on it in the other browser.
(And I then close that window and use the context menu on the original link leading)

- While I could use the extension button (the context menu seems more natural) I try and keep on screen clutter to a minimum, (so any extension button that I can put in the FF overflow pulldown goes there).

- For example the FF Wayback Machine extension can be used _anywhere on the background_ while viewing a page.

- To be able to have Open (This Tab) in Other Browser on background or the tab context menus would be a natural place to match the context menu shown hovering over a link.
- For these cases perhaps an alternative (or additional option toggle) to also _close the **current** tab_, effectively making it a "_Reopen In_ & Close" of the current page's URL in the other browser.

Many thanks for a very useful set of tools.

**Comment by Damon-debug** [comment]
excellent

**Comment by bruna17** [comment]
https://github.com/andy-portmen/open-in/issues/60#issuecomment-1033080004
